allensdk/core/brain_observatory_nwb_data_set.py

```diff
@@ -74,6 +74,11 @@ def get_epoch_mask_list(st, threshold, max_cuts=2):
     epoch_mask_list = []
 
     if len(cut_inds) > max_cuts:
+        
```

Comment:
Can you get rid of this debug plot?

Reply:
@nicain bump -- I'll merge this once you remove the extra print.

Reply:
added documentation of how to resolve exceptions like this (by pointing to the GH issue), and made the debug plot a commented-out snippet, so that I (or others) can use it if it ever happens again.  Merge if you feel good about this, or Ill cut it out completely.

Reply:
I vote to remove it.  You could make a custom exception that passes along the delta so that someone could plot it if they catch the exception, though.

Reply:
Is there a location in allensdk for custom Exceptions?

Reply:
https://github.com/AllenInstitute/AllenSDK/blob/master/allensdk/brain_observatory/brain_observatory_exceptions.py

Reply:
OK, I cut out the debugging plot, but moved all the data necessary to generate it into a special exception class that gets thrown.  I also documented how to perform this debugging in a gist (https://gist.github.com/nicain/bce66cd073e422f07cf337b476c63be7), and documented this gist and this issue with inline comments near where the exception is thrown.